src/scripts.js

```diff
@@ -277,50 +259,45 @@ favoriteRecipesView.addEventListener('click', event => {
     
 const displayFavoritedRecipes = () => {
   showFavoriteRecipesView();
-  favRecipeCards.innerHTML = ``
+  favoriteRecipesView.innerHTML = ``
   newUser.favoriteRecipes.forEach((recipe) => {
-    favRecipeCards.insertAdjacentHTML('afterBegin', 
+    favoriteRecipesView.insertAdjacentHTML('afterBegin', 
```

Comment:
I love the html change of doing just sections and inserting in the elements through the DOM.